src/components/Dashboard/Content/index.tsx

```diff
@@ -48,8 +49,8 @@ const DashboardContent = ({
   ) : (
     <div className="flex flex-col h-full" data-test-id={dashboardDataTestIds.content}>
       <div
-        style={{ backgroundImage: `url(${surveys[currentPosition].imageUrl})` }}
-        className="w-full h-[302px] rounded-[12px] bg-cover duration-500 ease-in-out"
+        style={{ backgroundImage: `url(${getHighResolutionImage(surveys[currentPosition].coverImageUrl)})` }}
+        className={`w-full h-[302px] rounded-[12px] bg-cover duration-500 ease-in-out`}
```

Comment:
Why do you need this change?

Reply:
@hoangmirs my bad, I forgot to change it back